<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>355</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>FotoYa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QPushButton" name="pbPublicador">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Publicador</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lHashtagActual">
     <property name="text">
      <string>Hash Tag:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="leHashtagActual">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pbProyectar">
     <property name="text">
      <string>Proyectar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QComboBox" name="cbHashtags"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pbCrearEvento">
     <property name="text">
      <string>Crear evento</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbWebInstagram">
       <property name="text">
        <string>Ver Instagram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbWebFacebook">
       <property name="text">
        <string>Ver Facebook</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbWebTwitter">
       <property name="text">
        <string>Ver Twitter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTimerScroll">
       <property name="text">
        <string>Timer Scroll (seg.)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTimerScroll">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTimerWeb">
       <property name="text">
        <string>Timer Web (seg.)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTimerWeb">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>60</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Impresor" name="impresor" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Galeria" name="galeria" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Impresor</class>
   <extends>QWidget</extends>
   <header>impresor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Galeria</class>
   <extends>QWidget</extends>
   <header>galeria.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
